api/__init__.py

```diff
@@ -0,0 +1,12 @@
+from Base import *
```

Comment:
Normally we don't have an init like this because it causes an wanted load.   Let's remove that and require them to import directly.  Just leave this empty.